server.go

```diff
@@ -197,11 +197,14 @@ type Server struct {
 	// Whether to set the SO_REUSEPORT socket option, allowing multiple listeners to be bound to a single address.
 	// It is only supported on go1.11+ and when using ListenAndServe.
 	ReusePort bool
-	// AcceptMsgFunc will check the incoming message and will reject it early in the process.
-	// By default DefaultMsgAcceptFunc will be used.
+	// AcceptMsgFunc will check the incoming message and will reject it early in the process, but after
+	// spawning a new go-routine. By default DefaultMsgAcceptFunc will be used.
 	MsgAcceptFunc MsgAcceptFunc
+	// PacketFunc will either allow the incoming packet, deny it, or have some in-between action, like sleeping a
+	// bit. DefaultPacketFunc can be used or serve as an example. This function is called for every 100th packet received.
+	PacketAccepter
```

Comment:
Although the name is the same, we don't really want to embed this, otherwise `Server` unexpectedly gains an `Accept` method.